### DISCUSSION TITLE
Re-inscribing an ordinal

### DISCUSSION THREAD

**Discussion by thomasgauthier** [ordinals/ord#1640] [Q&A]
When trying to re-inscribe a sat using  `ord --bitcoin-data-dir=/var/lib/bitcoind wallet inscribe --satpoint alreadyinscribedsatpoint` I get `error: sat at output:0:0 already inscribed`.

I imagine that's intended behavior but I do actually want to re-inscribe that sat. How can I do it? If I fork and remove the restriction and do it, will it show on the ordinals.com feed?

Edit : reinscription `2fbc91e6b9e5da25a12e0d2a29eea0cf2c93a9704c87b70f9b826b521fb27befi0` (commit `a2543387225d86bd38ad7107f449543f05e1eed676ecb086f4ab659e35c8f609`) doesn't show on ordinals.com

**Comment by raphjaph** [comment]
For now the first inscription is the canonical one. There are probably use cases for sub inscriptions but we haven't sketched those out yet. What kind of use cases do you have in mind?

**Reply by AhmetcanFR** [reply]
> For now the first inscription is the canonical one. There are probably use cases for sub inscriptions but we haven't sketched those out yet. What kind of use cases do you have in mind?

Hello Raphjaph ! It would be awesome to display reinscribed pictures at the front on ordinals.com. You asked for use cases but we cannot share informations on that in order to not be frontrunner on a project we're working, but you can believe that there is a lot of use cases for that one ! 

Would be awesome japh, thank you.
